packages/wdio-allure-reporter/src/utils.ts

```diff
@@ -105,3 +106,15 @@ export const getLinkByTemplate = (template: string | undefined, id: string) => {
     }
     return template.replace(linkPlaceholder, id)
 }
+
+/**
+ *
+ * @param {string} logs - logs to be attached
+ * @param {string} allure - allure report object
+ * @private
+ */
+export const attachConsoleLogs = (logs: string | undefined, allure: AllureReporter['_allure']) => {
+    if (logs) allure?.addAttachment('Console Logs',
+        '<pre style="display: inline-block; background-color:#4d4d4d;color:white;padding:20px;text-shadow:1px 1px 0 #444; min-width:100%; height:auto; min-height: 100%;">'
+            .concat('.........Console Logs.........\n\n'+logs, '</pre>'), 'text/html')
```

Comment:
```suggestion
    if (logs) {
        allure?.addAttachment(
            'Console Logs',
            '<pre style="display: inline-block; background-color: #4d4d4d; color: white; padding: 20px; text-shadow: 1px 1px 0 #444; min-width: 100%; height: auto; min-height: 100%;">'
                .concat('.........Console Logs.........\n\n'+logs, '</pre>'),
            'text/html'
        )
    }
```

Maybe rather than using `concat` just concat that with `+`